atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1955,7 +1955,7 @@ private void commitWrites(TransactionService transactionService) {
                 // if there are no writes, we must still make sure the immutable timestamp lock is still valid,
                 // to ensure that sweep hasn't thoroughly deleted cells we tried to read
                 if (validationNecessaryForInvolvedTablesOnCommit()) {
-                    preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired(null);
+                    preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired();
```

Comment:
In this case, if the user adds extra locks to the manager, we'll end up checking those. However, we see above that we ALSO check the preCommitConditions (let's think of those as AdvisoryLocks). So this doesn't immediately feel like a bad regression, and certainly should not cause any correctness issues: we're simply checking more locks potentially.

Reply:
Checked before and after and there are no functional changes. Correct, in the future we might check more things here as we add them to `TransactionLocksManager`. I do think we want to simplify this and maybe also kill `preCommitRequirementValidator` and loop it in `TransactionLocksManager` (in a flup PR).